source/manual/business-readiness-publish-changes.html.md

```diff
@@ -0,0 +1,40 @@
+---
+owner_slack: "#govuk-platform-health"
+title: Publishing changes to the business readiness finder
+section: Business readiness finder
+layout: manual_layout
+parent: "/manual.html"
+important: true
+last_reviewed_on: 2019-01-18
+review_in: 3 months
+---
+
+The full journey for the  [business readiness finder][business-readiness-finder] is made up of the content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The question pages: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+
+The content fo the start page can be updated in Mainstream Publisher as you would any other publisher.
```

Comment:
*for

Reply:
"as you would any other publisher" should this be "as you would any other start page"?

Reply:
Yes to both! Thank you!